Sources/FluentKit/Model/Model.swift

```diff
@@ -285,40 +268,3 @@ extension Model {
             .first()
     }
 }
-
-extension Model {
-    public func willCreate(on database: Database) -> EventLoopFuture<Void> {
```

Comment:
Interesting that you've dropped these. I guess it makes sense, they are redundant and less powerful than the middleware.